source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -620,6 +642,7 @@ def fill_journeys(self, request_type, api_request, instance):
 
         sort_journeys(pb_resp, instance.journey_order, api_request['clockwise'])
         tag_journeys(pb_resp)
+        journey_filter.delete_journeys((pb_resp,), api_request)
```

Comment:
The next calls to kraken would be create when we haven't yet deleted the filtered journeys, there is a risk that the next call will be based on a "invalid" journey.